option optimize_for = LITE_RUNTIME;

message LegionStickyMessageAnswer
{
 required uint32 result = 1;
 required uint32 message_id = 2;
}
